CHANGES.md

```diff
@@ -1,5 +1,136 @@
 # Changes
 
+## stable-2.12.0
+
+This release introduces route-based policy to Linkerd, allowing users to define
+and enforce authorization policies based on HTTP routes in a fully zero-trust
+way. These policies are built on Linkerd's strong workload identities, secured
+by mutual TLS, and configured using types from Kubernetes's [Gateway API](https://gateway-api.sigs.k8s.io/).
+
+The 2.12 release also introduces access logging, a long-awaited feature that
```

Comment:
Nit: the comma here makes these items sound separate, if we change it to a hyphen it's a little easier to follow.

```suggestion
The 2.12 release also introduces access logging — a long-awaited feature that
```

Reply:
Nice catch!